packages/fluentui/react-northstar/src/components/Pill/Pill.tsx

```diff
@@ -74,9 +76,31 @@ export interface PillProps extends UIComponentProps, ContentComponentProps<Short
    * @param data - All props.
    */
   onDismiss?: ComponentEventHandler<PillProps>;
+
+  /**
+   * A Pill can be selectable
+   */
+  selectable?: boolean;
+
+  /**
+   * A Pill state for selection
+   */
+  selected?: boolean;
```

Comment:
As this properly is controlled we should add a callback for it :)

Reply:
Done